sdk/examples/intkey_python/sawtooth_intkey/client_cli/workload.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import argparse
+import logging
+import random
+import threading
+import time
+from collections import namedtuple
+from datetime import datetime
+
+from sawtooth_signing import secp256k1_signer as signing
+from sawtooth_sdk.workload.workload_generator import WorkloadGenerator
+from sawtooth_sdk.workload.sawtooth_workload import SawtoothWorkload
+from sawtooth_sdk.client.stream import Stream
+from sawtooth_sdk.protobuf import batch_pb2
+from sawtooth_sdk.protobuf.validator_pb2 import Message
+from sawtooth_intkey.client_cli.create_batch import create_intkey_transaction
+from sawtooth_intkey.client_cli.create_batch import create_batch
+
+LOGGER = logging.getLogger(__name__)
+
+IntKeyState = namedtuple('IntKeyState', ['name', 'stream', 'value'])
+
+
+class IntegerKeyWorkload(SawtoothWorkload):
```

Comment:
Seems like the name for intkey should be standard. IntKey, IntegerKey, and intkey all appear in this file.